servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/DefaultLoadBalancerBuilder.java

```diff
@@ -167,7 +156,7 @@ public LoadBalancer<C> newLoadBalancer(
                     new XdsOutlierDetector<>(executor, outlierDetectorConfig, lbDescription);
             }
             return new DefaultLoadBalancer<>(id, targetResource, eventPublisher,
-                    DefaultHostPriorityStrategy::new, loadBalancingPolicy, subsetSize,
+                    DefaultHostPriorityStrategy::new, loadBalancingPolicy, new RandomSubsetter(Integer.MAX_VALUE),
```

Comment:
If we are not going to use a subsetter now, will it make sense to create a `NoopSubsetter` implementation that will simply return `hosts` as-is? It will be less code to execute inside LB if there is no need.

Reply:
We could, but it would be symbolic: `RandomSubsetter` checks if the desired subset is larger than the host set size and if so, it's a no-op. I'd like to keep that behavior even if we introduce a truly no-op subsetter.

Reply:
You are right, I think I misinterpreted the first `if` condition inside `RandomSubsetter`. All good.